test/pkg/environment/common/expectations.go

```diff
@@ -364,12 +364,30 @@ func (env *Environment) ExpectCreatedNodeCount(comparator string, count int) []*
 	return createdNodes
 }
 
+func MachineNames(machines []*v1alpha5.Machine) []string {
+	return lo.Map(machines, func(m *v1alpha5.Machine, index int) string {
+		return m.Name
+	})
+}
+
 func NodeNames(nodes []*v1.Node) []string {
 	return lo.Map(nodes, func(n *v1.Node, index int) string {
 		return n.Name
 	})
 }
 
+func (env *Environment) ConsistentlyExpectNodeCount(comparator string, count int, duration string) []*v1.Node {
+	GinkgoHelper()
+	By(fmt.Sprintf("waiting for nodes to be %s to %d", comparator, count))
```

Comment:
We need to make sure all of this messaging is consistent when we change the function to do something different. This `By` statement should say something like `expecting nodes to be %s to %d for %s", comparator, count, duration.String()`